.travis.yml

```diff
@@ -16,6 +25,18 @@ script:
     -e token
     my-test-image ./todo_app/test/test_trello_app.py
 
+after-script:
+  - docker push $DOCKER_USERNAME/todo-app:$TRAVIS_COMMIT
```

Comment:
This means that you're pushing your production image on every build, not just on the deployment branch (e.g. Module-8 currently) - this is unlikely to be what your team wants as you'd usually deploy only the master branch, i.e. after a change has passed review
Currently two developers working in tandem will be consistently pushing out their contradictory images
Moving this to the before_deploy stage would make more sense (and we could then move the build/tag steps there too, which would save some time in our CI pipeline for unmerged builds

Reply:
As advised I have moved it to before_deploy stage